src/main/java/org/vandeseer/easytable/drawing/DrawingUtil.java

```diff
@@ -21,6 +22,14 @@ public static void drawText(PDPageContentStream contentStream, PositionedStyledT
     }
 
     public static void drawLine(PDPageContentStream contentStream, PositionedLine line) throws IOException {
+
+        if(line.getBorderStyle() !=null && line.getBorderStyle() == BorderStyle.DOTTED){
```

Comment:
From my point of view the `BorderStyle` enum should contain what is specified here as parameters to the method call, i.e. the enum should look like:
    
    public enum BorderStyle {
        SOLID(new float[]{1}, 1), 
        DOTTED(new float[]{1}, 0)
        ...
    }

This way we don't need the `if`/`else` here and neither in other places.